graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/functions/strings/Concat.java

```diff
@@ -32,8 +32,8 @@ public class Concat extends AbstractFunction<String> {
     private final ParameterDescriptor<String, String> secondParam;
 
     public Concat() {
-        firstParam = ParameterDescriptor.string("first").description("First string").build();
-        secondParam = ParameterDescriptor.string("second").description("Second string").build();
+        firstParam = ParameterDescriptor.string("first").primary().description("First string").build();
+        secondParam = ParameterDescriptor.string("second").primary().description("Second string").build();
```

Comment:
`primary` was originally used to specify which parameters would consume output from the previous step. Since this has changed, `primary` is indeed not a good name anymore. Changing this to `ruleBuilderVariable`